apertium-eng.eng.rlx

```diff
@@ -328,7 +333,13 @@ SELECT (adv itg) IF (-1 BOS) ;
 SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
-
+SELECT Adv IF (0 Adj) (-1 Vblex) ;
+SELECT Pr IF (0 Cnjadv) (1 Det) ;
+SELECT Det IF (0 Adj) (1 Vblex) (2 N) ;
+SELECT Pr IF (0 Adv) (1 Np) ;
+SELECT Adv IF (0 N) (1 Adj) ;
+SELECT Cnjadv IF (0 Pr) (1 Prn) ;
```

Comment:
This will fail in an example like "...for me...", which is arguably more common.  You might be able to make it more specific to avoid that?

Reply:
I've done so. :)